src/calculateRentalCost.js

```diff
@@ -5,6 +5,23 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERN = 7;
+  const SHORT_TERN = 3;
+  const LONG_TERN_DISKOUNT = 50;
```

Comment:
There is a typo in the constant name 'LONG_TERN_DISKOUNT'. It should be 'LONG_TERM_DISCOUNT' to correctly reflect its purpose.